internal/db/db.go

```diff
@@ -186,15 +191,17 @@ func (db *dB) GetComposeImageType(jobId uuid.UUID, orgId string) (string, error)
 	return imageType, nil
 }
 
-func (db *dB) GetComposes(orgId string, since time.Duration, limit, offset int) ([]ComposeEntry, int, error) {
+func (db *dB) GetComposes(orgId string, since time.Duration, limit, offset int, IgnoreImageType []string) ([]ComposeEntry, int, error) {
 	ctx := context.Background()
 	conn, err := db.Pool.Acquire(ctx)
 	if err != nil {
 		return nil, 0, err
 	}
 	defer conn.Release()
 
-	result, err := conn.Query(ctx, sqlGetComposes, orgId, since, limit, offset)
+	// Then execute the query with the built sqlGetComposes string
+	result, err := conn.Query(ctx, sqlGetComposes, orgId, since, pq.Array(IgnoreImageType), limit, offset)
+
```

Comment:
The count also should be affected so we should also pass IgnorImageType to filter by, what do you think ?

Reply:
yes, correct, I added it as well, thank you